app/views/users/phone_setup/index.html.erb

```diff
@@ -37,20 +44,15 @@
       ).with_content(t('forms.buttons.send_one_time_code')) %>
 <% end %>
 
-<p class="margin-top-5">
-  <%= t('two_factor_authentication.phone_fee_disclosure') %>
-  <% if IdentityConfig.store.voip_block %>
-    <%= t('two_factor_authentication.two_factor_choice_options.phone_info_no_voip') %>
-  <% end %>
-</p>
-<% unless FeatureManagement.phone_recaptcha_enabled? %>
-  <%= link_to t('two_factor_authentication.mobile_terms_of_service'),
-              MarketingSite.messaging_practices_url,
-              class: 'display-block margin-bottom-4' %>
-<% end %>
+  <p class="margin-top-4">
+    <%= t(
+          'two_factor_authentication.recaptcha.disclosure_statement_html',
+          app_name: APP_NAME,
+          google_policy_link_html: new_tab_link_to(t('two_factor_authentication.recaptcha.google_policy_link'), GooglePolicySite.privacy_url),
+          google_tos_link_html: new_tab_link_to(t('two_factor_authentication.recaptcha.google_tos_link'), GooglePolicySite.terms_url),
+          login_tos_link_html: new_tab_link_to(t('two_factor_authentication.recaptcha.login_tos_link'), MarketingSite.rules_of_use_url),
+        ) %>
+  </p>
```

Comment:
We'll want to keep the feature flag consideration, right?

```suggestion
<% if FeatureManagement.phone_recaptcha_enabled? %>
  <p class="margin-top-4">
    <%= t(
          'two_factor_authentication.recaptcha.disclosure_statement_html',
          app_name: APP_NAME,
          google_policy_link_html: new_tab_link_to(t('two_factor_authentication.recaptcha.google_policy_link'), GooglePolicySite.privacy_url),
          google_tos_link_html: new_tab_link_to(t('two_factor_authentication.recaptcha.google_tos_link'), GooglePolicySite.terms_url),
          login_tos_link_html: new_tab_link_to(t('two_factor_authentication.recaptcha.login_tos_link'), MarketingSite.rules_of_use_url),
        ) %>
  </p>
<% end %>
```

Reply:
🤦🏾‍♀️ yes that feature flag should still be there